tools/firtool/firtool.cpp

```diff
@@ -165,143 +166,146 @@ static cl::opt<firrtl::PreserveValues::PreserveMode>
 
 static cl::opt<std::string>
     replSeqMemCircuit("repl-seq-mem-circuit",
-                      cl::desc("circuit root for seq mem metadata"),
+                      cl::desc("Circuit root for seq mem metadata"),
                       cl::init(""), cl::cat(mainCategory));
 
 static cl::opt<std::string>
     replSeqMemFile("repl-seq-mem-file",
-                   cl::desc("file name for seq mem metadata"), cl::init(""),
+                   cl::desc("File name for seq mem metadata"), cl::init(""),
                    cl::cat(mainCategory));
 
 static cl::opt<bool>
     ignoreReadEnableMem("ignore-read-enable-mem",
-                        cl::desc("ignore the read enable signal, instead of "
+                        cl::desc("Ignore the read enable signal, instead of "
                                  "assigning X on read disable"),
                         cl::init(false), cl::cat(mainCategory));
 
-static cl::opt<bool> imconstprop(
-    "imconstprop",
-    cl::desc(
-        "Enable intermodule constant propagation and dead code elimination"),
-    cl::init(true), cl::cat(mainCategory));
-
-static cl::opt<bool> lowerMemory("lower-memory",
-                                 cl::desc("run the lower-memory pass"),
-                                 cl::init(true), cl::cat(mainCategory));
+static cl::opt<bool>
+    disableIMCP("disable-imcp",
+                cl::desc("Disable the intermodule constant propagation pass"),
```

Comment:
Consistency nit: this option uses "intermodule" while the other uses "inter-module".

Also, IM{CP,DCE} and inliner are spelled out while mostly the rest are single-word pass names (and capitalized).  But "IMCP" and "IMDCE" might be a bit too opaque for the uninitiated, so sounds good to me.

Reply:
I went with "IMCP" and "IMDCE" in all capitals.